core/domain/email_manager.py

```diff
@@ -117,6 +131,13 @@ def log_new_error(*args, **kwargs):
     'I\'m writing to inform you that I have unpublished the above '
     'exploration.')
 
+NOTIFICATION_EMAILS_FOR_FAILED_TASKS = config_domain.ConfigProperty(
+    'notification_emails_for_failed_tasks',
+    NOTIFICATION_EMAIL_LIST_SCHEMA,
+    'Email(s) to notify if a training task fails',
```

Comment:
Email(s) to notify if an ML training task fails. (We can generalize this human-readable string later if we use the same property for other types of tasks, but for now let's just be explicit.)

Reply:
done